tests/BuildScriptGenerator.Tests/Python/PythonBashBuildSnippetTest.cs

```diff
@@ -186,5 +186,61 @@ public void GeneratedSnippet_ContainsPackageWheelType_When_PackageCommandAndPack
             Assert.Contains("Creating universal package wheel", text);
             Assert.Contains("setup.py sdist --formats=gztar,zip,tar bdist_wheel --universal", text);
         }
+
+        [Fact]
+        public void GeneratedSnippet_DisablePipUpgradeFlag_IfPipUpgradeFlag_IsNull()
```

Comment:
Technically the PipUpgradeFlag is empty, not null, so lets update this method name.